### ISSUE TITLE
Send command replies to direct messages

### ISSUE THREAD

**Issue by Gjum** [CivMC/Kira#14]
Currently, command replies (/ingame, /apitoken, etc.) are sent in the channel where the command is used, for anyone else to see. This often accidentally reveals private information (tokens, groups, etc.) of the command user. Example:
![image](https://github.com/CivMC/Kira/assets/2612164/b1d152e4-a7e1-419e-9929-65213d59567d)

One solution is to send all replies to the user's DMs.

Alternatively, the reply message should have the `ephemeral` flag, so only the command user can see it, and it disappears when the Discord app is restarted. Example:
![image](https://github.com/CivMC/Kira/assets/2612164/5a316dc9-0244-4094-bdb0-5e0e2083e87e)

Another option, for commands where it is desired to show the reply to others in the channel, would be to first reply with a button interaction "Do you want to show the reply in the channel here? [show in channel] [show in DMs]", and only send the actual reply to the channel or DMs accordingly.

**Comment by Liam-DeVoe** [comment]
I just want to throw out a consideration here, which is that [Snitchvis](https://github.com/tybug/snitchvisbot/) uses the output of `!kira relayconfig` to parse snitch messages correctly ([code](https://github.com/tybug/snitchvisbot/blob/019b443c4ccf88cd5e32b23a626a8a3192dbaf50/main.py#L1365C24-L1371)). If `!kira relayconfig` was dm'd instead, this would be considerably less user-friendly to set on snitchvis[^1]. Hopefully there will remain a way to send this information publicly. 

[^1]: Not that it's very friendly right now, mind you, but running `!kira relayconfig` followed by `.add-kira-config` is bearable enough.

**Comment by Gjum** [comment]
Yes, only potentially sensitive commands should go to DMs. Another example that should stay in the channel where it's used is `/whoami`, the response of which some bots read to easily link people's in-game and Discord accounts.
